src/python/pants/rules/core/BUILD

```diff
@@ -1 +1,17 @@
 python_library()
```

Comment:
Not directly related to your change, but FYI - this library target is missing `dependencies = []`  for the files it owns, now including your new file. This wasn't exposed until now because there were no tests here.  I think that for your tests to pass you will need to add the relevant deps here.